src/element/FollowsYou.css

```diff
@@ -0,0 +1,6 @@
+.follows-you {
```

Comment:
Very difficult to see 
![image](https://user-images.githubusercontent.com/1172179/213418868-899d88ae-7200-45fc-bfe0-57ad532b3cd8.png)